glue-codes/fast-farm/src/FASTWrapper_Types.f90

```diff
@@ -0,0 +1,2834 @@
+!STARTOFREGISTRYGENERATEDFILE 'FASTWrapper_Types.f90'
+!
+! WARNING This file is generated automatically by the FAST registry.
+! Do not edit.  Your changes to this file will be lost.
+!
+! FAST Registry
+!*********************************************************************************************************************************
+! FASTWrapper_Types
+!.................................................................................................................................
+! This file is part of FASTWrapper.
+!
+! Copyright (C) 2012-2016 National Renewable Energy Laboratory
+!
+! Licensed under the Apache License, Version 2.0 (the "License");
+! you may not use this file except in compliance with the License.
+! You may obtain a copy of the License at
+!
+!     http://www.apache.org/licenses/LICENSE-2.0
+!
+! Unless required by applicable law or agreed to in writing, software
+! distributed under the License is distributed on an "AS IS" BASIS,
+! WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+! See the License for the specific language governing permissions and
+! limitations under the License.
+!
+!
+! W A R N I N G : This file was automatically generated from the FAST registry.  Changes made to this file may be lost.
+!
+!*********************************************************************************************************************************
+!> This module contains the user-defined types needed in FASTWrapper. It also contains copy, destroy, pack, and
+!! unpack routines associated with each defined data type. This code is automatically generated by the FAST Registry.
+MODULE FASTWrapper_Types
+!---------------------------------------------------------------------------------------------------------------------------------
+USE ElastoDyn_Types
```

Comment:
@jjonkman Why are all of the module types included here and is it necessary? I don't see them used. This adds memory overhead when compiling so some automated systems fail (i.e. [conda-forge](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=305724&view=logs&jobId=656edd35-690f-5c53-9ba3-09c10d0bea97&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d)).

Reply:
This comes from the `FASTWrapper_Registry.txt` file:
```
usefrom FAST_Registry.txt
```
which is necessary to get definition of the `FAST_TurbineType` data type.

Reply:
Is there any support in the Registry to include `USE FAST_Types, ONLY: FAST_TurbineType` directly without using the whole FAST_Registry?

Reply:
If there isn't support for `only`, it would be very useful to add.

Reply:
@rafmudaf , there isn't currently support for that in the Registry. Have you tried to build with just a "USE FAST_Types" line instead of all the use statements from each module included in FAST_Types? If that fixes the memory issue that would be a **lot** easier to change in the Registry than trying to add an ONLY statement.

Reply:
Yes, I removed all of the unused module-types imports in both FAST_Farm_Types and FASTWrapper_Types and it did reduce the memory usage. Whether it fixes this particular problem (conda-forge CI runs out of memory compiling FAST_Farm_Types) I do not know.

Reply:
@rafmudaf I made some modifications to the registry this evening to get rid of the extra USE statements (ones that are inherited). Those changes are in my `f/Registry_Cleanup` branch. Hopefully that helps fix the issue because it would take a lot more thought to get USE ONLY functionality in the Registry.